src/maxminddb.c

```diff
@@ -750,6 +752,7 @@ static int populate_languages_metadata(MMDB_s *mmdb,
     for (uint32_t i = 0; i < array_size; i++) {
         member = member->next;
         if (MMDB_DATA_TYPE_UTF8_STRING != member->entry_data.type) {
+            MMDB_free_entry_data_list(first_member);
```

Comment:
Another good find! I think we will want to call this in all error paths in this function, e.g. on the blocks on line 748 and 762 would need it too I believe.

We should probably look at everything that is calling `MMDB_get_entry_data_list()` as I'd expect there to be more instances of this.

Reply:
> I think we will want to call this in all error paths in this function, e.g. on the blocks on line 748 and 762 would need it too I believe.

I didn't add it because I wasn't able to reproduce the two condition.

Reply:
> We should probably look at everything that is calling `MMDB_get_entry_data_list()` as I'd expect there to be more instances of this.

In the library, there are only two important calls for `MMDB_get_entry_data_list()`.